nidirect_workflow/src/Form/AuditSettingsForm.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
```

Comment:
Does PHPCS pass without a `@file` block comment?

Reply:
Yes it does, but will add a comment as discussed